runHZZFiducialXS.py

```diff
@@ -149,12 +150,13 @@ def extractBackgroundTemplatesAndFractions(obsName, observableBins, year):
     # Here, rm command is mandatory, just to ensure that make works. If make command files
     #   then this can pick older executable. So, to avoid this we first delete the executable
     logger.info("==> Remove the executable and Compile the package main_fiducialXSTemplates...")
-    cmd = 'rm main_fiducialXSTemplates; make';
+    #cmd = 'rm main_fiducialXSTemplates; make';
+    cmd = 'pwd';
```

Comment:
@qyguo 
 
Invert the change. The reason is already written in the code as a comment just before the command line:

```
    # Here, rm command is mandatory, just to ensure that make works. If make command files
    #   then this can pick older executable. So, to avoid this we first delete the executable
```